docs/milo/manual/chapters/20_introduction.md

```diff
@@ -22,62 +22,62 @@ Have fun building your very first Milo!
 
 The firmware used in this manual will only consist of the basic 3-axis functions of the machine.
 
-Touch probes, additional relays, toolsetters and other related add ons will not be part of this base firmware and therefore will not be present in this manual.
+Touch probes, additional relays, toolsetters, and other related add-ons will not be part of this base firmware and therefore will not be in this manual.
 
 ## RepRapFirmware
 
-RRF is typically considered a 3D printing firmware, but in the last few years has begun to see more and more CNC features implemented.
+RRF is typically considered a 3D printing firmware, but in the last few years, it has begun to see more and more CNC features implemented.
 
 RRF boards tend to operate over WiFi or Ethernet and do not stream gcode over the air, but rather load it onto the board before starting a machining operation. Keep this in mind when finding a place to store Milo, as you will need some form of connection to the control board to allow working with the machine itself.
 
 RRF is extremely control board agnostic, so for the most part firmware can be shared between different control boards with only a few pin definitions needing to be changed.
 
 Pins in RRF act a little differently from Klipper and Marlin, as almost all pins are given a pin number (ex PC_7)
-and a pin name (ex Xmin). It is generally the pin name that will be referenced in the firmware and not the pin number, meaning that Xmin on one control board will also be Xmin on others. Keep this in mind if you want to use a different control board from the one found in this manual.
+and a pin name (ex Xmin). It is generally the pin name that will be referenced in the firmware and not the pin number, meaning that Xmin on one control board will also be Xmin on others. Keep this in mind if you want to use a different control board from the one in this manual.
 
-It is still a good idea to cross reference these pins before changing any settings in firmware, and most control boards will come with a table similar to the one found [here](https://teamgloomy.github.io/fly_cdyv3_pins.html) - use this table in combination with a pinout diagram of your board to ensure everything will work as expected.
+It is still a good idea to cross-reference these pins before changing any settings in firmware, and most control boards will come with a table similar to the one found [here](https://teamgloomy.github.io/fly_cdyv3_pins.html) - use this table in combination with a pinout diagram of your board to ensure everything will work as expected.
 
 *[RRF]: RepRapFirmware
 
 ## Bill of Materials
 
-Provided [here](../../bom/sourcing_guide.md) is the bill of materials. Whilst we recommend that you try to stick to this list as much as possible, you're an adult (hopefully) and this is your machine. If there is a substitution that you think would lead to a better machine then feel free! If there is a feature you don't feel is necessary then don't buy the parts for it.
```

Comment:
whilst is the correct word

Reply:
This was addressed, however it didn't mark the request as outdated.

Reply:
eh, it's still choppy. Going to work on this one a touch more.